### PR TITLE
Rebuild for aarch64 and ppc64le

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #97

### CODE REVIEW

**Comment by rth** [comment]
Thanks for working on this @xhochy !

The  test failure on aarch64 in `test_estimate_bandwidth_1sample` can be ignored, cf https://github.com/scikit-learn/scikit-learn/pull/14187

Wow 2.4 hours to run the test suite, we really need to make it faster.

For ppc64le build looks like some connection issue to download dependencies?

**Comment by rth** [comment]
> Wow 2.4 hours to run the test suite, we really need to make it faster.

Though most of it is likely due to running ARM in QEMU.

**Comment by xhochy** [comment]
@rth Ignored test on the non-x86 arches and this passes now.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,7 +47,8 @@ test:
   commands:
     - set OPENBLAS_NUM_THREADS=1          # [win]
     - export OPENBLAS_NUM_THREADS=1       # [not win]
-    - pytest --verbose --pyargs sklearn
+    - pytest --verbose --pyargs sklearn  # [not (aarch64 or ppc64le)]
+    - pytest --verbose --pyargs sklearn -k 'not test_estimate_bandwidth_1sample'  # [aarch64 or ppc64le]
```
Comment (rth) [inline recipe/meta.yaml]: Very nice!

**Comment by xhochy** [comment]
@conda-forge/scikit-learn @conda-forge/arm-arch This is ready for review/merge.

**Comment by jakirkham** [comment]
@rth, would you like to be a maintainer here?

**Comment by jakirkham** [comment]
Can you send a PR adding yourself, @rth? Happy to merge 🙂

**Pull request merged**